nvtabular/ops/categorify.py

```diff
@@ -185,6 +186,11 @@ class Categorify(StatOperator):
         and our new translated dataframe entry will now be [[17], [17, 20], [19, 18], [18]].
         This parameter is useful to reserve an initial segment of non-negative translated integers
         for special user-defined values.
+    cardinality_memory_limit: int or str, default None
```

Comment:
Open to suggestions on the best name here. In this PR, I want to use this attribute as a guide for deciding whether or not to perform memory-optimization logic. However, in the future, we *should* be able to use it to choose better defaults across the board.

Reply:
That seems like a reasonable naming.